config/_default/config.toml

```diff
@@ -167,6 +167,13 @@ name = "Community"
     URL = "https://github.com/kubesphere/kubesphere/blob/master/docs/roadmap.md"
     weight = 5
 
+    [[languages.en.menu.main]]
+    parent = "Community"
+    name = "Summer 2021"
```

Comment:
This name might make people feel confused. Because a lot of people know Google Summer of Code. They might think this is GSoC.

So, how about using the following name: `Summer 2021 of OSPP`

OSPP means Open Source Promotion Plan

Reply:
good idea.